rules/windows/builtin/system/win_susp_service_installation_folder_pattern.yml

```diff
@@ -12,9 +13,8 @@ detection:
     Provider_Name: 'Service Control Manager'
     EventID: 7045
   suspicious1:
-    ImagePath|re:
-      - '^[Cc]:\\[Pp]rogram[Dd]ata\\.{1,9}\.exe'
-      - '^[Cc]:\\.{1,9}\.exe'
```

Comment:
What about this change - this shouldn't be necessary or is it? @thomaspatzke

Reply:
But it seems that this is also the only rule that contains a list of regular expressions described in that way. All other rules use 

```
- ImagePath|re: 'xxx'
- ImagePath|re: 'yyy'
```
instead of 
```
ImagePath|re:
   - 'xxx'
   - 'yyy'
```

Reply:
`re` need a string (can not be a list of string)
```
class SigmaRegularExpressionModifier(SigmaTypeModifier):
    """Treat value as regular expression"""
    identifier = "re"
    active = True
    valid_input_types = (str,)
```